demisto_sdk/commands/validate/validators/RM_validators/RM115_no_default_section_left.py

```diff
@@ -0,0 +1,90 @@
+from __future__ import annotations
+
+import re
+from typing import Iterable, List, Optional, Union
+
+from demisto_sdk.commands.content_graph.objects.integration import Integration
+from demisto_sdk.commands.content_graph.objects.pack import Pack
+from demisto_sdk.commands.content_graph.objects.playbook import Playbook
+from demisto_sdk.commands.content_graph.objects.script import Script
+from demisto_sdk.commands.content_graph.parsers.related_files import RelatedFileType
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Union[Integration, Script, Playbook, Pack]
+
+USER_FILL_SECTIONS = [
+    "FILL IN REQUIRED PERMISSIONS HERE",
+    "version xx",
+    "%%UPDATE%%",
+]
+PACKS_TO_IGNORE = ["HelloWorld", "HelloWorldPremium"]
+DEFAULT_SENTENCES = ["getting started and learn how to build an integration"]
+
+
+class NoDefaultSectionsLeftReadmeValidator(BaseValidator[ContentTypes]):
+    error_code = "RM115"
+    description = "Validate that no default section were left in the readme"
+    error_message = "Replace {0} with a suitable info."
```

Comment:
The following sections still contains the default template, make sure to fill them with the relevant information {0}.